app.js

```diff
@@ -0,0 +1,69 @@
+import express from "express";
+import "dotenv/config";
+
+import deleteMovie from "./queries/deleteMovie.js";
+import getAllMovies from "./queries/getAllMovies.js";
+import insertMovie from "./queries/insertMovie.js";
+import updateMovie from "./queries/updateMovie.js";
+
+const app = express();
+const port = 3000;
```

Comment:
I assume this was added before you switched to environment variables, but I would make sure to delete all code that doesn't have a purpose